text/0000-export-report.md

```diff
@@ -0,0 +1,76 @@
+# Meta
+[meta]: #meta
+- Name: Export Report
+- Start Date: (fill in today's date: 2020-04-01)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: "N/A"
+
+# Summary
+[summary]: #summary
+
+The lifecycle export stage should optionally write a machine parsable metadata "report" about the resulting exported image(s).
+
+# Motivation
+[motivation]: #motivation
+
+The logging output of the lifecycle export stage displays the built image's digest or image id. However, this information is not easily available for a platform that needs to utilize the resulting image from a build. 
+
+Without this information, a platform will need to resort to suboptimal mechanisms such as parsing the exported output or reading the most recently tagged image on the registry. 
+
+
+# What it is
+[what-it-is]: #what-it-is
+
+The lifecycle exporter/creator binary will accept an optional flag `exported` which will provides a path to `exported.toml`. When the flag is supplied the export stage will write a toml representation of the built image for platform consumption. When the flag is not provided it will default to writing exported.toml in the working directory which by convention is `/layers`.
+
+Here is an overview of the suggested schema for exported.toml:
+
+```
+[image]
+tags = ["index.docker.io/image/name:latest", "index.docker.io/image/name:other-tag"]
+identifier = "sha256:c8be1b8f4d60d99c281fc2db75e0f56df42a83ad2f0b091621ce19357e19d853"
```

Comment:
`identifier` appears to be generic. It would be nice to know if it's an Image ID (in the daemon case) or Digest (in the case of a registry).